.github/workflows/php.yml

```diff
@@ -12,11 +12,13 @@ jobs:
       fail-fast: true
       matrix:
         php: [8.0]
```

Comment:
Needs to add PHP 8.1 support too on the workflow.

Reply:
`PHP Fatal error:  During inheritance of IteratorAggregate: Uncaught ErrorException: Return type of Symfony\Component\HttpFoundation\ParameterBag::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/runner/work/Laraconfig/Laraconfig/vendor/symfony/http-foundation/ParameterBag.php:205`

looks like it also need newer of symfony
How about bump min version to [v8.74.0](https://github.com/laravel/framework/releases/tag/v8.74.0)?